workshops/weather_app/README.md

```diff
@@ -0,0 +1,409 @@
+API.
+
+```
+function showWeather(details){  //Taking the received values from API into this function
+}
+
+```
+
+Here we pass the `details` received from API to the `showWeather` function. 
+
+
+Before coding further we should try seeing what we obtain from the API using `console.log()`.
+
+```
+function showWeather(details){  //Taking the received values from API into this function
+
+    // console.log(details);
+}
+
+```
+Now we see the Console log of the browser to see the details we have received from the API. The console should look like:
+
+![console](https://cloud-1tlmm2zp9.vercel.app/0console.gif)
+
+After we have got to know the details received from API, we will now store them in a variable and push them to the HTML.
+
+```
+function showWeather(details){  //Taking the received values from API into this function
+
+    // console.log(details);
+
+    let city = document.getElementById('city');
+    city.innerHTML = `${details.name}, ${details.sys.country}`;
+
+    let temperature = document.getElementById('temp');
+    temperature.innerHTML = `${Math.round(details.main.temp)}&deg;C`; 
+
+    let minMax = document.getElementById('min-max');
+    minMax.innerHTML = `${Math.round(details.main.temp_min)}&deg;C (Min) and ${Math.round(details.main.temp_max)}&deg;C (Max) `; 
+
+    let weatherType = document.getElementById('weather-type');
+    weatherType.innerHTML = `${details.weather[0].main}`;
+}
+
+```
+
+Since we are using `JSON` for accessing the details, the format of getting the details is something like this `details.property` 
+where `details` is our `JSON` object and `property` could be any element of that object. For choosing the correct `property` we have to see
+the console log of the details we receive from the API.
+
+Also, to round off the temperature we are using `Math.round()` function.
+
+Yipeee! We have now finsihed coding our weather app and the App is now ready to be used.
+
+For seeing the end result use the `Run` button on `Repl`.
+
+![app](https://cloud-g1zuwwz9o.vercel.app/0app.gif)
+
+## 6. The Improvised Version
+
+Here are some things which you should consider to improvise your knowledge of APIs and make some changes in this project as well.
+
+- You could add different image/font-awesome icon depending upon the weather type(sunny, rainy etc).
+
+- Try to make another card on the left of this where you display the map of the city entered.(You will get to know about some new APIs of Maps).
+
+- There are many other details which we receive from the API(humidity,pressure etc) that we do not display. See what other details we receive
+and try to display them as well.
+
+- Add a `right-arrow` font-awesome icon which when clicked shows the details instead of pressing `Enter`. (Some basic JS practice)
+
+Last but not the least, be as creative and dynamic as possible. This is only the start and I am sure after this workshop you would definitely create

```

Comment:
You need to link to your or other hackers’ demos of doing something fun/different with the project—see guidelines